contrib/linux/dosbox-staging.metainfo.xml

```diff
@@ -0,0 +1,54 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<!-- Copyright 2020 DOSBox-staging team -->
+<component type="desktop">
+  <id>io.github.dosbox-staging.dosbox-staging</id>
+  <project_license>GPL-2.0-only</project_license>
+  <metadata_license>CC0-1.0</metadata_license>
+  <provides>
+    <id>io.github.dosbox-staging</id>
+  </provides>
+  <content_rating type="oars-1.1" />
+  <name>DOSBox-staging</name>
```

Comment:
@kcgen I think we should change our convention about putting name in human-readable places…

Packagers invent newer and newer ways of avoiding spelling "dosbox-staging", and it's a bit understandable - projects usually don't name themselves starting with a lower letter.

What do you think about migrating our naming convention to "DOSBox Staging" in human-readable contexts (but keeping "dosbox-staging" in all technical places of course).

@okias This is not a blocker issue for PR, I think me and @kcgen need to pick a new typographic scheme; if you want to change this field, then I suggest using "DOSBox Staging".

Reply:
`DOSBox Staging` sounds better, but honestly I must say that staging is for me associated with development wine-staging. Since it has similar story as dosbox & dosbox-staging, as I understood original DOSBox author(s) rejected merging stuff back (or doing anything :laughing: ). For car enthusiast you could call DOSBox Stage 2, but except joke I cannot offer better input.

Reply:
The program's name should be 

```Dosbox: To Hell with Vogons Edition```

(Joking)

Reply:
@MasterO2 TBH I like `:` there maybe `DOSBox: Staging` would look better... but that's not yet it... maybe we could go Elon's way `X Æ A-12`

Reply:
My suggestion: why not something like ```DOSBox Reloaded```, similar to ```D-Fend Reloaded```?

Reply:
@Wengier ATM we're not changing the name - there's mind-boggingly large number of places where naming needs to be adjusted; but the name "DOSBox Reloaded" is one of better suggestions I heard :)

Until now we suggested using "dosbox-staging" everywhere, but in practice I have seen users and packagers use any possible variation: DOSBox Staging, Dosbox Staging, Dosbox-Staging, DOSBox_staging, Dosbox-staging… every possible combination except the one we wanted to use :) Therefore I think we need to adjust.

Let's use "DOSBox Staging" in here (this seems to be the most common variation used by packagers, I think).

I will create PR with other places I feel need adjustment - it's out of scope of this PR of course.

Reply:
I'm OK with DOSBox Staging; I'm just worried that it's no longer a modification of the original name but instead fully uses the original name. 

So instead of a modified noun such as  _Sony-X_ and _sony-staging_, I would be more reluctant to start a company called _Sony <something>_.

I would want to explain our reason for this name adjustment to the DOSBox team to clarify this is due to confusion regarding lower-case proper names. 

That said; "DOSBox Staging" means that users will mis-case it just like they currently do in vogons.. Dosbox, DOSbox, DosBox, .. etc (which dosbox-staging has elegantly avoided).

Reply:
Reloaded is definitely a cool name :-)

In the Linux world, _staging_ has a powerful connotation - it not only holds the latest code, but also acts as a target or guide for downstream code or precursor versions. 

I might be wrong, but (my own impression) is that the use of '-staging' postfix also seems to coincide roughly in time with git and more modern SCM practices; which is one of the most important parts of our project (regardless of GitHub/gitlab / ..)

So for me, _staging_ carries a deeper meaning from the open source world.

(Versus -next, which was a postfix used by Linus's right-hand-man Andrew Morton's branch back in the BitKeeper days.)

Reply:
> Let's use "DOSBox Staging" in here (this seems to be the most common variation used by packagers, I think).

Sounds good @dreamer! I've sent an email to the DOSBox Team with this exact context to close the loop.

Reply:
@dreamer and @kcgen So I assume it is already decided. The term "DOSBox Staging" will be officially used from now on then.

Reply:
Someone can now fork DOSBox Staging and call it dosbox-staging :-)